readme.md

```diff
@@ -242,6 +242,7 @@
 - [llnode](https://github.com/nodejs/llnode) - Post-mortem analysis tool which allows you to inspect objects and get insights from a crashed Node.js process.
 - [thetool](https://github.com/sfninja/thetool) - Capture different CPU, memory, and other profiles for your app in Chrome DevTools friendly format.
 - [swagger-stats](https://github.com/slanatech/swagger-stats) - Trace API calls and monitor API performance, health, and usage metrics.
+- [Node.js V8 --inspector Manager (NiM)](https://github.com/june07/nim) - Streamline Your Debugging Workflow with Chromium (Chrome, Edge, More) DevTools.
```

Comment:
This doesn't actually describe what it does. Drop the marketing talk.

Reply:
Better?